### ISSUE TITLE
Don't know the loss curve of train process. Could you please share the loss curve ?

### ISSUE THREAD

**Issue by MrHsj** [hkchengrex/XMem#27]
Hi @hkchengrex！I trained on two 2060s, the batch size was set as 2, and only changed the size of batch_size. It was found that the loss curve decreased first, reached the lowest point of 1 at about 10K, and then kept rising until it reached 3 at 100K. Is this a normal phenomenon?
My loss curve is as follows:  https://img.danjiebetter.top/website/img/xmem_question1.png

**Comment by hkchengrex** [comment]
See https://github.com/hkchengrex/XMem/issues/26#issuecomment-1233109935

**Comment by MrHsj** [comment]
Thank you for your reply！The default stage 2's iteration is 100k. Do you mean that 10K is actually enough?

**Comment by MrHsj** [comment]
And I'm training a long training s3, which also reaches the lowest loss at about 10K, and then starts to rise, which makes me feel very strange

**Comment by MrHsj** [comment]
Thanks you again!

**Comment by MrHsj** [comment]
I'm sorry to bother you again. I found that my trained model can't get the same results as yours, which is worse than yours. When I use XMem-s01.pth for training stage 2, I can't get the best XMem-s012.pth. Except for the different batchsize, there is no change. Is there something wrong? There is the loss curve：https://img.danjiebetter.top/website/img/xmem_question2.png

**Comment by hkchengrex** [comment]
A batch size of 2 is quite a downscale from the default 8. You might need to tune the number of iterations and the learning rate.

**Comment by MrHsj** [comment]
Thank you, I'll try it.

**Comment by fiskrt** [comment]
Hey @MrHsj did you have any success in tuning the iters and lr for smaller batch sizes? I'm in the same situation but with batch size 4.
